.github/CODEOWNERS

```diff
@@ -244,15 +386,22 @@
 
 # Speed Improvement team.
 /app.yaml @vojtechjelinek
+/core/templates/dev/head/pages/Base.js @vojtechjelinek
 /core/templates/dev/head/pages/base.html @vojtechjelinek
+/core/templates/dev/head/pages/footer.html @vojtechjelinek
 /core/templates/dev/head/pages/footer_js_libs.html @vojtechjelinek
 /core/templates/dev/head/pages/header_css_libs.html @vojtechjelinek
 /core/templates/dev/head/pages/header_js_libs.html @vojtechjelinek
+/gulpfile.js @vojtechjelinek
+/jinja_utils*.py @vojtechjelinek
+/Makefile @vojtechjelinek
```

Comment:
@vojtechjelinek, is there any need of this file?  (Introduced at #5659)

Reply:
No, I was just playing with it and accidentaly commited the file.

Reply:
I've removed this file!